Makefile

```diff
@@ -615,7 +622,7 @@ realclean: clean java_clean
 	@$(RM) -f *.bin debian/changelog*
 
 ### additionally removes generated files not in the repository tree
-distclean: realclean clean_genRules
+distclean: clean_stack realclean clean_genRules
```

Comment:
If downloading is not allowed during launchpad building, then there seems to be no way to ensure this environment for the PPA... Even if we don't use Stack for the PPA and if manage to use the same GHC as in development *consistently*, we are stuck with the Debian-packaged Haskell-packages (e.g. glade) and cannot use the versions specified by the Hets.cabal and stack.yaml. This may lead to differences between the developed Hets and the Hets that users install via apt-get. This pull request is supposed to remove this difference.